vignettes/Creating_a_confusion_matrix.Rmd

```diff
@@ -97,7 +97,7 @@ basic_table <- table(d_binomial)
 basic_table
 ```
 
-In order to plot it with `ggplot2`, we convert it to a data frame with `broom::tidy()`:
```

Comment:
I don't think this one should be changed. Although, I see that this would require broom to build, so that needs a solution.

Reply:
This just needs to be replaced by `as_tibble`. Done!

``` r
library(tidyverse)

d_binomial <- tibble("target" = rbinom(100, 1, 0.7),
                     "prediction" = rbinom(100, 1, 0.6))

basic_table <- table(d_binomial)

broom::tidy(basic_table)
#> Warning: 'tidy.table' is deprecated.
#> See help("Deprecated")
#> # A tibble: 4 x 3
#>   target prediction     n
#>   <chr>  <chr>      <int>
#> 1 0      0              9
#> 2 1      0             24
#> 3 0      1             20
#> 4 1      1             47

as_tibble(basic_table)
#> # A tibble: 4 x 3
#>   target prediction     n
#>   <chr>  <chr>      <int>
#> 1 0      0              9
#> 2 1      0             24
#> 3 0      1             20
#> 4 1      1             47
```

<sup>Created on 2021-02-18 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0)</sup>

Reply:
Perfect!